README.md

```diff
@@ -1175,7 +1186,32 @@ POST /policies
       "service-offerings": [
         "p-config-server"
       ]
-    }
+    },
+    {
+    "resource-notification-policies": [
+        {
+            "resource-email-template": {
+                "from": "admin@pcf.demo.ironleg.me",
+                "subject": "Platform Updates",
+                "body": "Please take a moment to review the platform updates and share it with your Org users"
+            },
+            "resource-email-metadata": {
+                "resource": "organizations",
+                "labels": [
+                            "PrimaryOwner",
+                            "SecondaryOwner"
+                          ],
+                "domain": "pivotal.io"
```

Comment:
Please let's be explicit... update `domain` be `email_domain`.

Reply:
Done changed domain attribute to email-domain